backend/src/main.py

```diff
@@ -15,6 +15,26 @@ class UserRequest(BaseModel):
 # form_id -> fixed questions
 fixed_questions_store = {}
 
+class QuestionConfig(BaseModel):
+	followup_depth: int
+	criteria: List[str]
+
+class Question(BaseModel):
+	question: str
+	question_config: QuestionConfig
+
+class FormRequest(BaseModel):
+	form_id : int
+	questions : List[Question]
```

Comment:
We can create a different class instead of duplicating code here

Will refactor in my PR